test/test_torch.py

```diff
@@ -3605,6 +3605,32 @@ def test_unbiased(self):
         self.assertEqual(tensor.std(), tensor.std(unbiased=True))
         self.assertEqual(tensor.std(unbiased=False), tensor.std(0, unbiased=False)[0])
 
+    @unittest.skipIf(not torch.cuda.is_available(), 'no CUDA')
+    def test_unbiased_cuda(self):
+        tensor = torch.randn(100).cuda()
+        self.assertEqual(tensor.var(0), tensor.var(0, unbiased=True))
```

Comment:
This looks like it's just testing that `unbiased=True` is the default. That's not very useful.

Test that the outputs match the CPU functions or that it matches some known value.

Since you are improving numerical precision, add a test for values that you know would cause unacceptable numerical errors using the previous method.